_pages/decide/task-flow-analysis.md

```diff
@@ -6,27 +6,31 @@ redirect_from: /task-flow-analysis/
 
 ## What it is
 
-A step-by-step analysis of a common task a user must perform that diagrams the various touch points and decision points a user goes through to accomplish the task. The touch points should be represented as steps taken by the user, as well as steps taken by the system.
+A step-by-step analysis of how a user will interact with a system in order to reach a goal. This analysis is documented in a diagram that traces a user's possible paths through sequences of tasks and decision points in pursuit of their goal. The tasks and decision points should represent steps taken by the user, as well as steps taken by the system.
```

Comment:
Good call. I will edit. While I think task flow diagrams do not always have to include steps taken by "the system,"  they should include system actions, when systems are involved. So, will edit the system out of the first sentence, and revise the last sentence to be more clear.